packages/components/fold/index.ts

```diff
@@ -0,0 +1,8 @@
+import { App } from "vue"
+import fold from "./index.vue"
+
+export default {
+  install: (app: App<Element>) => {
+    app.component("fold", fold)
+  }
+}

```

Comment:
**Request:** prefer `signleQuote` than `doubleQuote`, also use `withInstall` under `@element-plus/utils/with-install` for exporting this.

Please refer to other components to see how this works.